### ISSUE TITLE
Emoticons forces ReplayMod to render nametags.

### ISSUE THREAD

**Issue by ImEthanBradberry** [mchorse/emoticons#13]
When trying to render a movie, for some reason when using the Emoticons mod, Replaymod ignores when you uncheck the "Render Name Tags" box, so it just renders the nametag even if you don't want to.

[img]https://i.johni0702.de/6wTqZ.png[/img]

This box is unchecked, so the nametag shouldn't be visible when you render the movie, but like I said before, Emoticons for some odd reason, ignores this and always renders the nametag, which ruins machinimas.

Since ReplayMod is the easiest way to do a simple timelapse, and Emoticons is such an awesome addition to Minecraft (I can't play Minecraft anymore without Emoticons) it would be great if we can finally enjoy both mods at once.

Versions being used are:

Minecraft 1.12.2
replaymod-1.12.2-2.3.0.jar
emoticons-0.4.3-1.12.2-dist.jar
mclib-1.0.4-1.12.2.jar

I spent an insane amount of time trying to pinpoint the exact mod that was creating this problem and turns out it's Emoticons so I hope this is looked into. Thanks

**Comment by mchorse** [comment]
Yeah, this issue was already reported, and I have it on my todo list to fix. I don't have any ETA on when it's going to get fixed, as I'm working hard on my other mod updates.

> Emoticons for some odd reason, ignores this and always renders the nametag, which ruins machinimas.

Honestly, if you're making a machinima, you should rather use [Blockbuster](https://www.youtube.com/playlist?list=PLLnllO8nnzE-LIHZiaq0-ZAZiDO82K1I9) and [Aperture](https://www.youtube.com/playlist?list=PLLnllO8nnzE8MGDb6QzE2kt4-KVC1dRRl) mods.

**Comment by ImEthanBradberry** [comment]
> 
> 
> Yeah, this issue was already reported, and I have it on my todo list to fix. I don't have any ETA on when it's going to get fixed, as I'm working hard on my other mod updates.
> 
> > Emoticons for some odd reason, ignores this and always renders the nametag, which ruins machinimas.
> 
> Honestly, if you're making a machinima, you should rather use [Blockbuster](https://www.youtube.com/playlist?list=PLLnllO8nnzE-LIHZiaq0-ZAZiDO82K1I9) and [Aperture](https://www.youtube.com/playlist?list=PLLnllO8nnzE8MGDb6QzE2kt4-KVC1dRRl) mods.

Is the issue located? I always wondered why Emoticons screwed up nametags.

I would like to use Blockbuster and Aperture but they seem too complicated to me to be honest. ReplayMod is more straightforward, specially for a one shoot construction timelapse. I have tried looking at your tutorials and I can't get my head around it and understand the workflow.

ReplayMod's has many problems that could be improved upon tho. You can't for instance have instant transitions from camera to camera, so I have to put 2 keyframes next to each other and trim any additional frames picked between said 2 keyframes in Vegas which is ridiculous. It also reloads the world when I fastforward the timeline or backwards which is annoying.

I should really give Aperture+Blockbuster combo a try but everytime I tried I ended up frustrated and just went to ReplayMod, since im just doing some simple stuff anyway where I need some exterior cameras.

**Comment by mchorse** [comment]
No, the issue wasn’t located, as I said, I’m working on other stuff, so you would have to wait until I’m done, and that’s at least a month, sorry. 😓

**Comment by ImEthanBradberry** [comment]
I waited years to be able to use both mods at the same time, i guess I can wait another month without going insane 🤪
